.vscode/settings.json

```diff
@@ -34,5 +34,6 @@
         "sagemath",
         "Cython"
     ],
-    "editor.formatOnType": true
+    "editor.formatOnType": true,
+    "esbonio.sphinx.confDir": ""
```

Comment:
What does this change do? Does it have a relation to the goal of this PR?

Reply:
Vscode is adding this every time I'm opening the project. I was bound to commit it at some point ;-). I doesn't hurt...